app/controllers/TaxAccountSummaryController.scala

```diff
@@ -76,19 +76,31 @@ class TaxAccountSummaryController @Inject()(trackingService: TrackingService,
           }
   }
 
-  private def taxAccountSummaryViewModel(nino: Nino, taxAccountSummary: TaxAccountSummary)(implicit hc: HeaderCarrier, messages: Messages) = {
+  private def taxAccountSummaryViewModel(nino: Nino, taxAccountSummary: TaxAccountSummary)
+                                        (implicit hc: HeaderCarrier, messages: Messages): Future[TaxAccountSummaryViewModel] =
     for {
-      taxCodeIncomes <- taxAccountService.taxCodeIncomes(nino, TaxYear())
+      livePensionIncomeSources <- taxAccountService.incomeSources(nino, TaxYear(), "PensionIncome", "Live")
+      liveEmploymentIncomeSources <- taxAccountService.incomeSources(nino, TaxYear(), "EmploymentIncome", "Live")
+      ceasedEmploymentIncomeSources <- taxAccountService.incomeSources(nino, TaxYear(), "EmploymentIncome", "Ceased")
+      nonMatchingCeasedEmployments <- employmentService.ceasedEmployments(nino, TaxYear())
```

Comment:
EmploymentIncome and PensionIncome should not be strings, I think they already align to TaxCodeComponentTypes? Live and Ceased are defined as TaxCodeIncomeSourceStatus so use these if possible